### PR TITLE
ashift : enhance a little tooltips

### PR DESCRIPTION
**Pull request by AlicVB**
- indicate the right click+drag possibility in the slider tooltip
- same in the mouse action in the help window
- indicate the minimum number of lines needed in the log message in case of error

as usual, feel free to correct the spelling ;)
esp. the rotation slider tooltip. I've copy-paste what was in c&r, but that sound strange to me !

### CODE REVIEW

File: src/iop/ashift.c [hunk 1]
```diff
@@ -5698,7 +5700,7 @@ void gui_init(struct dt_iop_module_t *self)
 
   self->widget = main_box;
 
-  gtk_widget_set_tooltip_text(g->rotation, _("rotate image"));
+  gtk_widget_set_tooltip_text(g->rotation, _("right-click and drag a line on the image to drag a straight line"));
```
Comment (elstoc) [inline src/iop/ashift.c:5703]: Should still reference the functionality of the slider, plus slight wording change. How about...

```suggestion
  gtk_widget_set_tooltip_text(g->rotation, _("rotate image\nright-click and drag to define a horizontal or vertical line by drawing on the image"));
```

**Review by TurboGit — APPROVED** [review]
Works for me, thanks !

**Comment by TurboGit** [comment]
Not directly related but I just found an issue:
1. add a vertical line and an horizontal one using manual pencil
2. click on correct vertical perspective (first button)
3. message displayed : 2 minimum control line : OK
4. quit darktable
5. restart it
6. open the same image
7. click on manual pencil
8. click on correct vertical perspective
9. the correction is done, yet we still have a single line in both direction!

(let me know if you prefer that I create a new issue)

**Comment by AlicVB** [comment]
@TurboGit : I can't reproduce this issue. Can you confirm that it still happen with all the recent changes ? Thanks !

**Comment by TurboGit** [comment]
@AlicVB : No this has been fixed by another commit, don't remember which one but we I saw it I was pretty sure it would fix this issue and I tested it... All was OK.

**Pull request merged**